Source/WTF/wtf/PlatformJSCOnly.cmake

```
@@ -1,3 +1,15 @@
+if (WTF_CPU_X86_64)
+    if (MSVC)
```

You could fold the two checks into a single `if`, as follows:

```suggestion
if (WTF_CPU_X86_64 AND MSVC)
```

---

It was copy pasta from `PlatformWin.cmake` which should probably be doing that as well.